src/shell/focus/target.rs

```diff
@@ -497,8 +497,8 @@ impl TouchTarget<State> for PointerFocusTarget {
                 TouchTarget::down(window, seat, data, event, seq)
             }
             PointerFocusTarget::StackUI(stack) => TouchTarget::down(stack, seat, data, event, seq),
-            // TODO: implement TouchTarget for ResizeFork/Grabs
-            PointerFocusTarget::ResizeFork(_fork) => {}
+            // TODO: implement TouchTarget for Grabs
```

Comment:
Is this comment still relevant with the latest changes?

Reply:
Now that all the grabs should have touch support, I guess that comment isn't really applicable anymore.

Though handling touch events on SSDs to start drags is still needed.

Reply:
> Though handling touch events on SSDs to start drags is still needed.

Is that something you want to address before I go ahead and merge this?

Reply:
It's probably not that hard to add, but it can be done later.

Reply:
My cosmic-comp changes for https://github.com/Smithay/smithay/pull/1386 are also based on this PR.